config-linux.md

```diff
@@ -494,6 +494,49 @@ You MUST specify at least one of the `hcaHandles` or `hcaObjects` in a given ent
 }
 ```
 
+## <a name="configLinuxUnified" />Unified
+
+**`unified`** (object, OPTIONAL) allows cgroup v2 parameters to be modified at runtime for the container.
+
+Each key in the map refers to a file in the cgroup unified hierarchy.
+
+The OCI runtime MUST ensure that the needed cgroup controllers are enabled for the cgroup.
+
+Configuration unknown to the runtime MUST still be written to the relevant file.
+
+The runtime MUST generate an error when the configuration refers to a cgroup controller that is not present or that cannot be enabled.
+
+### Example
+
+```json
+"unified": {
+    "io.max": "259:0 rbps=2097152 wiops=120\n253:0 rbps=2097152 wiops=120",
+    "hugetlb.1GB.max": "1073741824"
+}
+```
+
+If a controller is enabled on the cgroup v2 hierarchy but the configuration is provided for the cgroup v1 equivalent controller, the runtime MAY attempt a conversion.
+
+If the conversion is not possible the runtime MUST generate an error.
+
+### Example
```

Comment:
Any chance to add the complete conversion convention (either as an example or as a part of the spec)?

Reply:
should I copy the table from https://github.com/containers/crun/blob/master/crun.1.md#cgroup-v2 ?

Reply:
yes please

Reply:
added